pyexchange/bittrex.py

```diff
@@ -174,25 +93,21 @@ def place_order(self, pair: str, is_sell: bool, price: Wad, amount: Wad) -> str:
         assert(isinstance(price, Wad))
         assert(isinstance(amount, Wad))
 
-        params = {
-            "quantity": str(amount),
-            "rate": str(price),
-            "market": pair
+        body = {
+            'marketSymbol': pair,
+            'direction': "SELL" if is_sell else "BUY",
+            'quantity': float(price),
+            'limit': float(price),
```

Comment:
did you experiment turning this into a string rather than a float? float can cause some decimal issues.